.github/workflows/release.yml

```diff
@@ -0,0 +1,57 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "[0-9]+.[0-9]+.[0-9]+"
```

Comment:
This probably (incorrectly) allows for zero-prefixed numbers (the CLI also likely does that currently if I remember correctly).

Tested `^([1-9][0-9]*|[0-9]).([1-9][0-9]*|[0-9]).([1-9][0-9]*|[0-9])$` briefly which seems like it'd do what we're after (in here as well as the CLI code), depending on the regex syntax in use of course.

Example: https://regex101.com/r/dy9h3O/1

Reply:
We might be able to work around this but it'll be ugly... give me a minute

Reply:
Assuming it accepts multiple match conditions, we could do
```yml

on:
  push:
    tags:
      - "[0-9].[0-9].[0-9]"
      - "[0-9].[0-9].[1-9][0-9]*"
      - "[0-9].[1-9][0-9]*.[0-9]"
      - "[0-9].[1-9][0-9]*.[1-9][0-9]*"
      - "[1-9][0-9]*.[0-9].[0-9]"
      - "[1-9][0-9]*.[0-9].[1-9][0-9]*"
      - "[1-9][0-9]*.[1-9][0-9]*.[0-9]"
      - "[1-9][0-9]*.[1-9][0-9]*.[1-9][0-9]*"
```
😄

Reply:
Alternatively, we could try our darndest to not push tags with leading zeroes? 🤷🏻‍♂️

Reply:
Perhaps we could make the pipeline fail if the tag is wrong? That should be easier (and you could probably use the single-line regex mentioned above)

Reply:
Sure, I'll look into it.